examples/go/main.go

```diff
@@ -1,9 +1,8 @@
 package main
 
 import (
-	"github.com/pulumi/Azure-Container-Registry-Geo-Replication"
+	"github.com/pulumi/pulumi-azure-quickstart-acr-geo-replication/sdk/go/azure"
```

Comment:
We probably don't want to use the `azure` namespace. 
```suggestion
	acr "github.com/pulumi/pulumi-azure-quickstart-acr-geo-replication/sdk/go/azure"
```

Reply:
This is related to the codegen issue I opened at https://github.com/pulumi/pulumi/issues/8140